release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.h.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 4be821ace4f6e414596aefd815c62414
+guid: 013d47fff23e843e18f828636fa4ad16
```

Comment:
The original meta files were always changed by Unity, so I had to commit this change. Once changed, they don't get changed again.